### PR TITLE
austin.intro.1.maura.brady

### PR DESCRIPTION
**Pull request by maurab013**

### CODE REVIEW

File: 02Lesson/01Project/index.html [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
+      <li>Tom Hughes</li>
+    </ol>
+    <h4>
+      1912 World Series
+    </h4>
+    <p>
+      Another world series
+    </p>
+  </body>
 </html>
```
Comment (zacacollier) [inline 02Lesson/01Project/index.html:43]: You are writing some really clean HTML, keep it up!

File: 02Lesson/01Homework/index.html [hunk 2]
```diff
@@ -1,8 +1,129 @@
...
-<body>
+
+<body style="background-color:lightblue">
+  <h1 style="text-align:center">
+    MoBrady MoProblems
+  </h1>
+  <p style="text-align:center">
+    <em>You can sleep when you're dead</em>
+  </p>
+    <div style="text-align:center">
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:15]: Indentation: After closing `p` tag the following opening tag should start on the same level

File: 02Lesson/01Homework/index.html [hunk 3]
```diff
@@ -1,8 +1,129 @@
...
+      <img src="https://scontent-dft4-1.xx.fbcdn.net/v/t1.0-9/10262186_10204371733316536_8640703755058897068_n.jpg?oh=4d29b309382bed855da92ee7ed585681&oe=5891FF36" alt="Disco Ball" height="250" width="250"/>
+    </div>
+  <h2 style="font-family:fantasy">
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:18]: Indentation: After closing `div` tag, the following opening tag needs to be on the same level

File: 02Lesson/02Homework/index.html [hunk 4]
```diff
@@ -3,24 +3,24 @@
...
+  <main>
+    <section>
       <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
       <h2>Welcome to Git</h2>
       <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. </p>
-    </div>
-    <div class="section">
+    </section>
+  </main>
       <div class="article">
```
Comment (ivome) [inline 02Lesson/02Homework/index.html:24]: Indentation: After closing `main` tag, the next opening element must be on the same indentation level

File: 02Lesson/02Homework/index.html [hunk 5]
```diff
@@ -37,24 +37,24 @@
         <p>You do not need to subscribe: you will be Cc'd in replies. Please keep the Cc list intact when replying (use "Reply to all"). Greylisting may delay your first post for a few hours. Note also that the list only accepts plain-text email; please disable HTML in your outgoing messages.</p>
         <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on Google Groups which is a nice place for beginners to ask about anything.</p>
       </div>
-    </div>
-    <div id="aside">
+
+    <aside>
```
Comment (ivome) [inline 02Lesson/02Homework/index.html:41]: Indentation... see above 😄

File: 02Lesson/02Project/css/style.css [hunk 6]
```diff
@@ -1,24 +1,59 @@
+
 body {
-  width: 75%;
-  margin: 0 auto;
+  background-image:url("http://www.randco.com/media/wysiwyg/RCo-PDP-Background-Images/outerspace.jpg");
```
Comment (ivome) [inline 02Lesson/02Project/css/style.css:3]: Add a space after the colon of the CSS property.

File: 02Lesson/02Project/css/style.css [hunk 7]
```diff
@@ -1,24 +1,59 @@
...
-  width: 20%;
+  width: 30%;
+  height:auto;
+  background-color: white;
+  text-align: center;
 }
 
 footer {
   clear: both;
+  background-color:#FAF38C;
```
Comment (ivome) [inline 02Lesson/02Project/css/style.css:61]: Always add spaces after the `:` for better readability.

File: 02Lesson/02Project/index.html [hunk 8]
```diff
@@ -1,10 +1,43 @@
...
+        <p>
+          Brought to you by <a href="http://www.ufoevidence.org/">UFO Evidence</a>
+        </p>
+      </aside>
+    </main>
+    <footer>
+      <p>
+        &copy; Daily Planet, 2016
+      </p>
+    </footer>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:55]: Nicely formatted code here and good use of semantic elements!

**Review by ivome — COMMENTED** [review]
This is great work! I just commented on some very minor formatting and indentation issues. The code for the last project is perfect. Keep up the good work!

File: 02Lesson/01Homework/index.html (hunk 2 above)
Comment (maurab013) [reply 02Lesson/01Homework/index.html:15]: Ahh yes, thank you!

File: 02Lesson/02Project/css/style.css (hunk 6 above)
Comment (maurab013) [reply 02Lesson/02Project/css/style.css:3]: Okay... so in this case there should be a space between background-image: and url but the width and margin are correctly formatted?

Comment (zacacollier) [reply 02Lesson/02Project/css/style.css:3]: yep, should be `background-image: url("http://www.randco.com/media/wysiwyg/RCo-PDP-Background-Images/outerspace.jpg");`

**Pull request closed without merging**